pantheon/src/main/java/tech/pegasys/pantheon/controller/MainnetPantheonController.java

```diff
@@ -147,7 +147,7 @@ private MainnetPantheonController(
 
     final TransactionPool transactionPool =
         TransactionPoolFactory.createTransactionPool(
-            protocolSchedule, protocolContext, ethProtocolManager.ethContext());
+            protocolSchedule, protocolContext, ethProtocolManager.ethContext(), Clock.systemUTC());
```

Comment:
Would be good if we could only create one clock instance in the controller (there's a second one being passed to `DefaultBlockScheduler`.

Reply:
done -- we now have a single call to `Clock.systemUTC()` in the codebase.